functions/brinks/src/middleware/eventProcessedCheck.js

```diff
@@ -0,0 +1,26 @@
+// verify that this stripe event id has not yet been processed.
+import debug from 'debug';
+import isEventProcessed from '../lib/isEventProcessed';
+
+const dlog = debug('that:api:functions:brinks:eventProcessedCheckMw');
+
+export default function eventProccessedCheck(req, res, next) {
+  dlog('eventProccessedCheck called');
+  const { thatBrinks, stripeEvent } = req;
+  thatBrinks.stages.push('eventProcessCheck');
```

Comment:
should we make these 'stages' more of a constant type of thing? I've now noticed it in a few places.

Reply:
Constant? I am not following. They are what they are for that file.
It is a way we track which middleware has been visited to aid in troubleshooting. This data will be in Sentry.